src/witan/app/demo_data.clj

```diff
@@ -3,7 +3,7 @@
             [witan.app.data :as data]
             [witan.app.forecast :as forecast]
             [witan.app.model :as model]
-            [witan.app.user :as user]))
```

Comment:
I agree expressive names are better, problem is that  `user` is special because leiningen aliases it in a strange way. This was a quick fix, there may be a more involved fix, but I find that grudgingly accepting leiningen's defaults is usually the easiest.